src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -29,6 +29,52 @@ class TestHostComponent {
     @Input() pageState: DotRenderedPageState = mockDotRenderedPageState;
 }
 
+@Injectable()
+class MockDotLicenseService {
+    isEnterprise(): Observable<boolean> {
+        return of(true);
+    }
+}
+
+@Component({
```

Comment:
Mocking components it could lead to side effects, if we change something, like removing an `@Input()` of tny of this component, the tests here will not break.